### PR TITLE
WIP: Feature/duet edit

### LINKED ISSUE #30: Option to create duet/multi vocal songs
I'd love to have an option to create a duet song (2 or more vocals (see https://github.com/performous/performous/issues/235)) with composer.
Currently the only option to do this is by starting a new project, and create a txt specific for the 2nd or 3rd singer. Then copying those lines back into the original one and add the tags P1, P2 (and in the future maybe PX where X is equal to the amount of mics we have connected)

### PR DESCRIPTION
**Pull request by nieknooijens**
added in a feature to support dual track edit.

# closes #30

### CODE REVIEW

File: ui/editor.ui [hunk 1]
```diff
@@ -470,7 +493,7 @@
     </property>
     <widget class="QMenu" name="menuPreferences">
      <property name="title">
-      <string>P&amp;references</string>
+      <string>Pr&amp;eferences</string>
```
Comment (Baklap4) [inline ui/editor.ui]: Hmmm i think the ampersand shouldn't be here?

File: ui/editor.ui [hunk 2]
```diff
@@ -625,7 +648,7 @@
   </action>
   <action name="actionLyricsToFile">
    <property name="text">
-    <string>Lyrics to fi&amp;le...</string>
+    <string>L&amp;yrics to file...</string>
```
Comment (Baklap4) [inline ui/editor.ui]: Hmmm i think the ampersand shouldn't be here?

File: ui/editor.ui [hunk 3]
```diff
@@ -758,7 +781,7 @@
   </action>
   <action name="actionAboutQt">
    <property name="text">
-    <string>About Qt...</string>
+    <string>About &amp;Qt...</string>
```
Comment (Baklap4) [inline ui/editor.ui]: Hmmm i think the ampersand shouldn't be here?

File: ui/editor.ui [hunk 4]
```diff
@@ -768,7 +791,7 @@
   </action>
   <action name="actionLyricsFromLRCFile">
    <property name="text">
-    <string>Timed lyrics from LRC/Soramimi file...</string>
+    <string>&amp;Timed lyrics from LRC/Soramimi file...</string>
```
Comment (Baklap4) [inline ui/editor.ui]: Hmmm i think the ampersand shouldn't be here?

**Comment by Baklap4** [comment]
Will take a look at the other files once i'm back home since i can compile it then etc..

**Comment by nieknooijens** [comment]
It's not completely finished yet. I should add some context menu functions
for moving notes between tracks.
Also, there's a bug that the first note you create in an empty file gets
dumped into both tracks for some reason.

2019年6月11日(火) 21:57 Arjan Spieard <notifications@github.com>:

**Comment by Baklap4** [comment]
Converted to Draft PR as it's not finished yet :)

**Comment by Baklap4** [comment]
Going to close the PR as it's been left abandoned. Please reopen when new changes are ready

**Pull request closed without merging**